templates/default/rt/rtticketinfobox.html

```diff
@@ -349,9 +362,12 @@
 
 			<br>
 
-			{if $ticket.owner != $layout.logid}
+			{if !$ticket.owner && $ticket.owner != $layout.logid}
```

Comment:
A gdy ktoś jest już przypisany to nie będzie widać guzika przypisania do mnie. Dlaczego?
Podziel to na co najmniej 2 PR-y, bo obsługa nowego ustawienia konf. już by mogła pójść.

Reply:
> A gdy ktoś jest już przypisany to nie będzie widać guzika przypisania do mnie. Dlaczego?

Czy ładnie tak komuś zabierać zgłoszenia nie rozmawiając z nim o tym?
Mogę dorzucić superusera tutaj w warunki.

Reply:
Dodałem superusera.

Reply:
Dodany superuser.
PS. Nadal można to zrobić z ticketedit ale jest to utrudnione.